examples/custom-service.mdx

```diff
@@ -20,71 +38,154 @@ edition = "2021"
 [lib]
 
 [dependencies]
-shuttle-service = { version = "0.9.0" }
-shuttle-shared-db = { version = "0.9.0", features = ["postgres"] }
-sqlx = { version = "0.6", features = ["runtime-tokio-native-tls", "postgres"] }
+anyhow = "1.0.62"
+axum = "0.6.4"
+hyper = "0.14.24"
+poise = "0.5.2"
+serde = "1.0"
+shuttle-secrets = "0.10.0"
+shuttle-service = { version = "0.10.0"}
+sync_wrapper = "0.1.2"
+tokio = "1.22.0"
 ```
 
-This section will explain how to create a custom shuttle service. To do this we need 
-to return a struct from our `shuttle_service::main` function that implements `shuttle_service::Service`.
+### Getting Started
+To get started, we need to return a wrapper struct from our `shuttle_service::main` function that 
+implements `shuttle_service::Service`, which we can declare like so:
 
-In this simple example we will implement `Service` for a `PgPool`, that will simply return 'Hello world'
-from the database on startup. In Rust we can't implement foreign traits for foreign types, so we have 
-to create a wrapper around our `PgPool`.
-
-```rust
-struct PoolService {
-    pool: sqlx::PgPool,
+```rust lib.rs
+pub struct CustomService {
+    discord_bot:
+        FrameworkBuilder<Data, Box<(dyn std::error::Error +
+         std::marker::Send + Sync + 'static)>>,
+    router: SyncWrapper<Router>,
 }
 ```
 
 Then we need to implement `shuttle_service::Service` for our wrapper. If you need to bind to an address,
 for example if you're implementing service for an HTTP server, you can use the `addr` argument from `bind`.
 You can only have one HTTP service bound to the `addr`, but you can start other services that don't rely on 
-binding to a socket. A common usecase is starting an HTTP server like `axum`, alongside a `serenity` bot 
-that doesn't need to bind to a socket. Then you would include both in the same wrapper struct, and start 
-them as normal in the `bind` function.
+binding to a socket.
+
+We can do do this by simply adding an implementation for our wrapper struct containing a function called `start`,
+which will contain all of our tasks that we want to run and simply run them all together in one `tokio::select!` macro
+(currently, this code doesn't work due to no base implementation - we will implement this later on):
```

Comment:
We just need to update this now that we don't have `start` anymore

Reply:
LGTM, merging